packages/core/src/browser/shell/tab-bar-toolbar.tsx

```diff
@@ -153,7 +153,12 @@ export class TabBarToolbar extends ReactWidget {
         const menuPath = ['TAB_BAR_TOOLBAR_CONTEXT_MENU'];
         const toDisposeOnHide = new DisposableCollection();
         for (const [, item] of this.more) {
-            toDisposeOnHide.push(this.menus.registerMenuAction([...menuPath, item.group!], {
+            if (item.group!.indexOf('/') > -1) {
+                const split = item.group!.split('/');
+                split.splice(2);
```

Comment:
In the case that the split has `2` elements, the result of the `splice(2)` would be empty.
Do we need extra handling to determine that the `split` operation has a length of 3 (subgroup)?

For instance, is there a case that it would not follow the pattern `group/submenu/subgroup`?

Reply:
Reworked it in the latest fixup.